errlist/fielderr.go

```diff
@@ -4,14 +4,14 @@ import "fmt"
 
 type FieldErr struct {
 	Field string `json:"field"`
-	Err   error  `json:"err"`
+	Err   string `json:"err"`
```

Comment:
To not override `Marshal` :)
Because `error` is marshaled without call of `Error()` method.